ports/libsimpleservo/api/build.rs

```diff
@@ -46,7 +46,7 @@ fn main() {
         if target.contains("windows") {
             println!("cargo:rustc-link-lib=libEGL");
         } else {
-            println!("cargo:rust-link-lib=EGL");
+            println!("cargo:rustc-link-lib=EGL");
```

Comment:
Thanks for the fix! You could also fix the comment above which refers to `rust-link-lib` as well.

Reply:
Ah, I didn't notice that. Fixed now.